doc/contributors-guide/contributing-guidelines.md

```diff
@@ -1,5 +1,11 @@
 # ℹ Contributing guidelines
 
+{% hint style="info" %}
+"Hi there :wave: Thank you for your interest to contribute to the Git and Github training resource".
+To find guidelines for contributing to the training resource, to know more follow the links below.
+{% endhint %}
+
+
```

Comment:
@LucianeFernandesRoque is this ready for review? Or  you are still working on it? Not sure, but I'll still leave a review anyway.

Nice job getting how to add a hint and where to add it.

But I have some comments:
- Use this text in the hint I stead (you are not to add the double quotes. The quotes I'm adding is just supposed to help you know that is what you should add): "Hi there 👋 Thank you for your interest to contribute to the Git and Github training resource. Find the guidelines for contributing to the training resource in the links below."
- In the issue, you can see that the "Git and github training resource" text is a link that leads somewhere. You didn't make that text a link.
- I also added a Note at the end of the issue for where you can find the guidelines for submitting your changes. Please check that guideline again and fix your pull request title. It will help me to track changes better.

Please make the above changes.